keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -63,7 +63,8 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 This document proposes adding a new `--subresource` flag to the following kubectl
 subcommands: `get`, `patch`, `edit`, `apply` and `replace`. The goal of this flag
-is to simplify the process of fetching and updating `status` and `scale` subresources.
+is to simplify the process of fetching and updating `status`, `scale` and `resize`
```

Comment:
Would it be better if we add resize KEP in kep.yaml references?

Reply:
You mean adding https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources ?

Reply:
Updated.